product_docs/docs/pem/8/pem_ha_setup/setup_ha_using_efm.mdx

```diff
@@ -235,6 +235,15 @@ This ensures that webserver is configured on the standby and is disabled by defa
     $ sudo chmod 640 /root/.pem/agent1.crt
 ```
 
+9. Disable & Stop Httpd & PEM Agent services if they are running on all replica nodes.
+
+```text
+systemctl disable httpd
+systemctl disable pemagent
+systemctl stop httpd
+systemctl stop pemagent
+```
```

Comment:
@traghavv @asheshv, is the order of commands correct here? Do you think we should have stop command before disable command?

Reply:
Yes, first stop and disable. 
```
# systemctl stop pemagent
# systemctl stop httpd
# systemctl disable pemagent
# systemctl disable httpd
```